dask_geopandas/utils.py

```diff
@@ -0,0 +1,21 @@
+def _calculate_mid_points(bounds):
+    """
+    Calculate geohash based on the mid points of each geometry
+
+    Parameters
+    ----------
+    gdf : GeoDataFrame
```

Comment:
This docstring does not match the implementation

Reply:
Updated in d284934241cf02e493a69c138fad2a65b43d7983